arduino/include/arduino.h

```diff
@@ -0,0 +1,180 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include "serialib.h"
+#include <ios>
+#include <thread>
+
+class Arduino
+{
+public:
+  enum HEADERS
```

Comment:
You need to separate between command and message type. I mean you ask the Arduino for something and get float from the Arduino